opennms-doc/guide-admin/src/asciidoc/text/config-files/opennms.properties.adoc

```diff
@@ -0,0 +1,920 @@
+
+[[file-opennms-properties]]
+=== File: opennms.properties
+
+This properties file allows you to set global properties for the opennms application.
+These properties will be set as system properties at the beginning of Java startup.
+Any properties set on the Java command-line with `-Dproperty=value` will be overridden by these values.
+
+[options="autowidth"]
```

Comment:
This is right now a more complicated topic and should be moved to a different topic for the reason it implies all tables in the doc we use right now and has implication on the PDF rendered output.